agency/README.md

```diff
@@ -176,28 +180,22 @@ Query Parameters:
 | Parameter | Type | Required/Optional | Description |
 | ----- | ---- | ----------------- | ----- |
 | `provider_id` | UUID | Required | A UUID for the Provider, unique within MDS |
-| `service_area_id` | UUID  | Optional | If provided, retrieve a specific service area (e.g. a retired or old service area). If omitted, will return all active service areas. |
 
-Response:
+Response: 
+
+The response is an order  JSON / List with each item has the following members. 
 
 | Field | Types  | Required/Optional | Other |
 | ----- | ---- | ----------------- | ----- |
 | `service_area_id` | UUID | Required |  |
-| `service_start_date` | Unix Timestamp | Required | Date at which this service area became effective |
-| `service_end_date` | Unix Timestamp | Required | Date at which this service area was replaced. If currently effective, place NaN |
-| `service_area` | MultiPolygon | Required | |
-| `prior_service_area` | UUID | Optional | If exists, the UUID of the prior service area. |
-| `replacement_service_area` | UUID | Optional | If exists, the UUID of the service area that replaced this one |
-| `type` | Enum | Required |  See [area types](#area-types) table |
-
-### Area types
-
-| `type` | Description |
-|------------| ----------- |
-| unrestricted | Areas where devices may be picked up/dropped off. A provider's unrestricted area shall be contained completely inside the agency's unrestricted area for the provider in question, but it need not cover the entire agency unrestricted area. See the provider version of the service areas endpoint |
-| restricted | Areas where device pick-up/drop-off is not allowed |
-| preferred_pick_up | Areas where users are encouraged to pick up devices |
-| preferred_drop_off | Areas where users are encouraged to drop off devices |
+| `service_start_date` | Unix Timestamp | Required | Datetime at which this service area enters into effect. For the base service_area, the time should be the start of permitted operations in the City. |
```

Comment:
I thought the spec was standardizing on milliseconds everywhere, not seconds.  I realize millis is overkill for this, but the spec should use the same notion of `timestamp` everywhere.